services/app-web/src/hooks/useStatePrograms.ts

```diff
@@ -8,7 +8,7 @@ const useStatePrograms = (): Program[] | [] => {
 
     if (loggedInUser && loggedInUser.__typename === 'StateUser') {
         statePrograms = loggedInUser.state.programs
-    } else {
+    } else if (loggedInUser && loggedInUser.__typename === 'CMSUser') {
```

Comment:
I made this more restrictive because otherwise `undefined` loggedInUser from tests were getting caught here and flooding logs with unnecessary print statements